ProcessOut/Classes/ProcessOut.swift

```diff
@@ -297,7 +313,7 @@ public class ProcessOut {
     ///   - invoiceId: Invoice generated on your backend
     ///   - token: Card token to be used for the charge
     ///   - handler: Custom 3DS2 handler (please refer to our documentation for this)
-    public static func makeCardPayment(invoiceId: String, token: String, handler: ThreeDSHandler) {
+    public static func makeCardPayment(invoiceId: String, token: String, handler: ThreeDSHandler, with: UIViewController) {
```

Comment:
Does that mean engineers will need to change their code?

Reply:
If their integration was based on the last release (`makeCardPayment`) yes they will have to add this context